tests/onnx/test_vaious_input_type.py

```diff
@@ -0,0 +1,120 @@
+"""
```

Comment:
The name of the file contains a typo. 
`vaious` -> `various`. 

Anyway, this file should have been probably better called `test_infer_input_shape.py`, since this is the functionality you are testing here.

Reply:
Renamed to `test_infer_input_shape.py`.